tracefs/tracefs.go

```diff
@@ -68,7 +68,16 @@ func ReadFile(relname string) ([]byte, error) {
 	return os.ReadFile(filepath.Join(root, relname))
 }
 
-// OpenFile opens the relative path provided, using the detected root of tracefs or debugfs
+// Open opens the relative path provided (similar to os.Open), using the detected root of tracefs or debugfs
+func Open(relname string) (*os.File, error) {
```

Comment:
`os.Open` is just this:
```
func Open(name string) (*File, error) {
	return OpenFile(name, O_RDONLY, 0)
}
```

Do you want to do the same here?

Reply:
Yeah I could do that, but it feels like extracting the internal of the stdlib that could change (I don't see why it would, but it feels like it could)

Reply:
The documentation does explicitly mention `O_RDONLY` so I don't think they could change it. 😉

Reply:
Ok let's do this then :)